src/functTest/groovy/com/bmuschko/gradle/docker/DockerSpringBootApplicationPluginFunctionalTest.groovy

```diff
@@ -0,0 +1,302 @@
+package com.bmuschko.gradle.docker
+
+import spock.lang.Requires
+import spock.lang.Unroll
+
+class DockerSpringBootApplicationPluginFunctionalTest extends AbstractFunctionalTest {
+    private static final String PROJECT_NAME = 'spring-boot'
+    private static final String CUSTOM_BASE_IMAGE = 'openjdk:7-alpine'
```

Comment:
Might consider changing to `openjdk:8-alpine` like what you did with the `DockerJavaApplicationPluginFunctionalTest` if only to keep things the same across the board. Might make sense to create a single property used by all tests.

Reply:
done